client/my-sites/checkout/checkout/composite-checkout.js

```diff
@@ -640,9 +640,22 @@ export default function CompositeCheckout( {
 					},
 				} );
 				debug(
-					'Domain contact info validation ' + ( data.messages ? 'errors:' : 'successful' ),
-					data.messages
+					'Domain contact info validation for domains',
+					domainNames,
+					'and contact info',
+					contactDetails,
+					'result:',
+					data
 				);
+				if ( ! data ) {
+					showErrorMessage(
```

Comment:
I thought we were no supposed to show this error notice anymore?

Reply:
This error will only happen if something goes really wrong, (which it did without 1eb53aa) but I figured explicitly showing an error would be better than a having a fatal stop the JS process. Or am I misunderstanding the question?

Reply:
Sorry, should have been clearer (and I probably misunderstood the code) - I was referring to the floating error notice that shows up when there's a validation error in the contact info. I probably jumped to conclusion that this is the reason it's showing up.

Reply:
Oh, got it! The validation notice is the (already existing) code block below this one. It's a good question as to if we should display it or not. In theory the editable fields should display the relevant validation errors, so this notice is extraneous. On the other hand, it's possible that the user might not see those inline notifications (eg: because of a screen reader) or that they might not be visible (eg: if the validation is triggered when the step is not active), in which case it would be wise to show this notice also. This is a design question that I'm not sure I'm qualified to answer, but I'd rather err on the side of more notifications than fewer for right now. We do have an [open issue to hide these notices shortly after they are displayed](https://github.com/Automattic/wp-calypso/issues/40220) which may make the issue of them being annoying less problematic.